ansible_base/migrations/_managed_definitions.py

```diff
@@ -0,0 +1,100 @@
+from django.conf import settings
+
+from ansible_base.rbac.permission_registry import permission_registry
+
+import logging
+
+
+logger = logging.getLogger('awx.ansible_base.migrations._managed_definitions')
```

Comment:
The `ansible_base` is a top-level package, it doesn't seem to be nested in `awx` package. In the existing codebase loggers do not use `awx` prefix.

Reply:
P.S. is there a reason why you don't use `__name__` special variable? It doesn't look like logger name distinct from module's qualified name.

Reply:
Thanks for the comments! I implemented all of them except the `__name__` suggestion, because I don't currently see this being done this django-ansible-base project. I would support that change on a whole-project basis, but for the purposes of this feature I want to keep consistency.